doc/source/language/flow.rst

```diff
@@ -494,56 +715,87 @@ Avoid looping during ``ON`` code blocks if you can. If you are going to make ext
 
 When a ``WHEN``/``THEN`` or ``ON`` condition is triggered, the default behavior is to execute the code block body exactly once and only once, and then the trigger condition is removed and the trigger will never occur again.
 
-To alter this, execute the ``PRESERVE`` command anywhere within the body of the code being executed and it tells the **kOS** computer to keep the trigger condition active. When it finishes executing the code block of the trigger, if ``PRESERVE`` has happened anywhere within that run of the block of code, it will not remove the trigger. Instead it will allow it to re-trigger, possibly as soon as the very next tick. If the ``PRESERVE`` keyword is executed again and again each time the trigger occurs, the trigger could remain active indefinitely.
+To alter this, a new ability was added in kOS 0.19.3 and above to
+have triggers simply :ref:`return a true or false value <trigger_return>`
+to determine if they wish to be preserved.
 
-The following example sets up a continuous background check to keep looking for if there's no fuel in the current stage, and if there is, then it activates the next stage, but no more often than once every half second. Once more than ``NUMSTAGES`` have happened, it allows the check to stop executing but it keeps the check alive until that happens::
+But prior to kOS 0.19.2 the only way to do it in kerboscript was
```

Comment:
good point.  My brain was thinking with 'less than or equal to' logic, but that's not how I wrote it.